addons/l10n_gr_edi/models/account_move.py

```diff
@@ -0,0 +1,718 @@
+from lxml import etree
+from werkzeug.urls import url_quote_plus
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.tools import cleanup_xml_node
+from odoo.addons.l10n_gr_edi.models.classification_data import (
+    CLASSIFICATION_CATEGORY_EXPENSE,
+    COMBINATIONS_WITH_POSSIBLE_EMPTY_TYPE,
+    INVOICE_TYPES_HAVE_EXPENSE,
+    INVOICE_TYPES_HAVE_INCOME,
+    INVOICE_TYPES_SELECTION,
+    PAYMENT_METHOD_SELECTION,
+    TYPES_WITH_CORRELATE_INVOICE,
+    TYPES_WITH_FORBIDDEN_COUNTERPART,
+    TYPES_WITH_FORBIDDEN_QUANTITY,
+    TYPES_WITH_MANDATORY_COUNTERPART,
+    TYPES_WITH_MANDATORY_PAYMENT,
+    TYPES_WITH_VAT_CATEGORY_8,
+    TYPES_WITH_VAT_EXEMPT,
+    VALID_TAX_AMOUNTS,
+)
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_gr_edi_mark = fields.Char(
+        string='MyDATA Mark',
+        compute='_compute_from_l10n_gr_edi_document_ids',
+        store=True,
+    )
+    l10n_gr_edi_cls_mark = fields.Char(
+        string='MyDATA Classification Mark',
+        compute='_compute_from_l10n_gr_edi_document_ids',
+        store=True,
+    )
+    l10n_gr_edi_document_ids = fields.One2many(
+        comodel_name='l10n_gr_edi.document',
+        inverse_name='move_id',
+        copy=False,
+        readonly=True,
+    )
+    l10n_gr_edi_state = fields.Selection(
+        selection=[
+            ('invoice_sent', 'Invoice sent'),
+            ('invoice_error', 'Invoice send failed'),
+            ('bill_fetched', "Expense classification ready to send"),
+            ('bill_sent', "Expense classification sent"),
+            ('bill_error', "Expense classification send failed"),
+        ],
+        string='MyDATA Status',
+        compute='_compute_from_l10n_gr_edi_document_ids',
+        store=True,
+        tracking=True,
+    )
+    l10n_gr_edi_available_inv_type = fields.Char(compute='_compute_l10n_gr_edi_available_inv_type')
+    l10n_gr_edi_correlation_id = fields.Many2one(
+        comodel_name='account.move',
+        string='MyDATA Correlated Invoice',
+    )
+    l10n_gr_edi_inv_type = fields.Selection(
+        selection=INVOICE_TYPES_SELECTION,
+        string='MyDATA Invoice Type',
+        compute='_compute_l10n_gr_edi_inv_type',
+        store=True,
+    )
+    l10n_gr_edi_payment_method = fields.Selection(
+        selection=PAYMENT_METHOD_SELECTION,
+        string='MyDATA Payment Method',
+        default='1',
```

Comment:
Make it a computed field and set a value only for GR country. We don't want a non-empty column for all others localization